script/trading/seaport/DeployImmutableSignedZoneV2.s.sol

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Immutable Pty Ltd 2018 - 2024
+// SPDX-License-Identifier: Apache-2
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.20;
+
+import "forge-std/Test.sol";
+import {ImmutableSignedZoneV2} from "../../../contracts/trading/seaport/zones/immutable-signed-zone/v2/ImmutableSignedZoneV2.sol";
+import "forge-std/console.sol";
```

Comment:
If I understand correctly, this `import` isn't required if we've already imported `forge-std/Test.sol`

Reply:
I should've removed the console logs all together, will update

Reply:
[bc5640c](https://github.com/immutable/contracts/pull/215/commits/bc5640c3e5c1748f0e28a3d84a267fd49d7fd2f4)